VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -30,13 +29,12 @@
     @ColorInt
     private final int edx_utility_success_dark;
 
-    @Inject
-    public DiscussionPostsAdapter(Context context, IEdxEnvironment environment) {
-        super(context, R.layout.row_discussion_thread, environment);
-        edx_brand_primary_base = context.getResources().getColor(R.color.edx_brand_primary_base);
-        edx_grayscale_neutral_light = context.getResources().getColor(R.color.edx_grayscale_neutral_light);
-        edx_brand_secondary_dark = context.getResources().getColor(R.color.edx_brand_secondary_dark);
-        edx_utility_success_dark = context.getResources().getColor(R.color.edx_utility_success_dark);
+    public DiscussionPostsAdapter(Fragment fragment, IEdxEnvironment environment) {
+        super(fragment.getContext(), R.layout.row_discussion_thread, environment);
```

Comment:
On the other hand, inflation is already being handled in the super implementation, so we might as well use the application Context everywhere, and just have that passed directly into the constructors without needing to change the implementation.